sdk/textanalytics/ai-text-analytics/swagger/README.md

```diff
@@ -78,37 +79,17 @@ directive:
 ```yaml
 directive:
   - from: swagger-document
-    where: $.definitions
-    transform: >
-      if (!$.TextDocumentStatistics) {
-          $.TextDocumentStatistics = $.DocumentStatistics;
-          delete $.DocumentStatistics;
-      }
-  - from: swagger-document
-    where: $.definitions..properties.statistics
-    transform: >
-      if ($["$ref"] && $["$ref"] === "#/definitions/DocumentStatistics") {
-          $["$ref"] = "#/definitions/TextDocumentStatistics";
-      }
+    where: $.definitions.DocumentStatistics
```

Comment:
We're missing `transform: >` between where and the transform. This should look like this:

```yaml
directive:
  - from: swagger-document
    where: $.definitions.DocumentStatistics
    transform: >
        $["x-ms-client-name"] = "TextDocumentStatistics";
```

This is true for the other transforms, make sure all of them have `transform: >`